test/cli/detailed-errors-completeness/test.out

```diff
@@ -0,0 +1,261 @@
+test.rb:65: The `lower` type bound `[Lower]` must be a supertype of the parent's `lower` type bound `[Middle]` for type_member `X` https://srb.help/5053
+    65 |  X = type_member {{upper: [Upper], lower: [Lower] }}
+                                                   ^^^^^^^
+    test.rb:47: `A::X` defined in parent here
+    47 |  X = type_member {{upper: [Middle], lower: [Middle] }}
+          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Possible explanation:
+    `Middle` is not a subtype of `Lower` for item `0`
+  Autocorrect: Use `-a` to autocorrect
+    test.rb:65: Replace with `[Middle]`
+    65 |  X = type_member {{upper: [Upper], lower: [Lower] }}
+                                                   ^^^^^^^
+
+test.rb:65: The `upper` type bound `[Upper]` must be a subtype of the parent's `upper` type bound `[Middle]` for type_member `X` https://srb.help/5053
+    65 |  X = type_member {{upper: [Upper], lower: [Lower] }}
+                                   ^^^^^^^
+    test.rb:47: `A::X` defined in parent here
+    47 |  X = type_member {{upper: [Middle], lower: [Middle] }}
+          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Possible explanation:
+    `Upper` is not a subtype of `Middle` for item `0`
+  Autocorrect: Use `-a` to autocorrect
+    test.rb:65: Replace with `[Middle]`
+    65 |  X = type_member {{upper: [Upper], lower: [Lower] }}
+                                   ^^^^^^^
+
+test.rb:93: The `fixed` type bound `[Middle]` must be equivalent to the parent's `fixed` type bound `[Upper]` for type_member `X` https://srb.help/5053
+    93 |  X = type_member {{ fixed: [Middle] }}
+                                    ^^^^^^^^
+    test.rb:87: `C::X` defined in parent here
+    87 |  X = type_member {{ fixed: [Upper] }}
+          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Possible explanation:
+    `Upper` is not a subtype of `Middle` for item `0`
+  Autocorrect: Use `-a` to autocorrect
+    test.rb:93: Replace with `[Upper]`
+    93 |  X = type_member {{ fixed: [Middle] }}
+                                    ^^^^^^^^
+
+test.rb:99: The `lower` type bound `[Upper]` is not a subtype of the `upper` type bound `[Lower]` for `E::X` https://srb.help/5052
+    99 |  X = type_member {{upper: [Lower], lower: [Upper] }}
+              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Possible explanation:
+    `Upper` is not a subtype of `Lower` for item `0`
+
+test.rb:68: Parameter `x` of type `[String]` not compatible with type of overridable method `A#foo` https://srb.help/5035
+    68 |  def foo(x, y:, z: [''], &blk)
+          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    test.rb:50: The super method parameter `x` was declared here with type `[Integer]`
+    50 |  def foo(x, y:, z: [1], &blk)
+          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Note:
+    A parameter's type must be a supertype of the same parameter's type on the super method.
```

Comment:
By convention, the `Note` should always be last in Sorbet error messages. If you could audit the places where you've added `e.addErrorSections` to make sure it's below any `addErrorNote` (**and** below any helper function which might eventually call `addErrorNote`) that would be good.